test/Microsoft.ML.Functional.Tests/ONNX.cs

```diff
@@ -51,15 +58,9 @@ public void SaveOnnxModelLoadAndScoreFastTree()
             var onnxEstimator = mlContext.Transforms.ApplyOnnxModel(modelPath);
             var onnxModel = onnxEstimator.Fit(data);
 
-            // TODO #2980: ONNX outputs don't match the outputs of the model, so we must hand-correct this for now.
-            // TODO #2981: ONNX models cannot be fit as part of a pipeline, so we must use a workaround like this.
```

Comment:
>// TODO #2981: [](start = 12, length = 14)

Maybe keep this TODO in until it's fixed?

Reply:
@rogancarr  - Thanks! I ended up fixing this, see the changes in GetOutputSchema in OnnxTransform.